src/main/java/com/ceritoxi/euler9/Euler9.java

```diff
@@ -0,0 +1,33 @@
+package com.ceritoxi.euler9;
+
+import com.ceritoxi.util.Math;
+
+public class Euler9 {
+	
+	private int result;
+	
+	public Euler9() {
+		this.result = task();
+	}
+	
+	private int task() {
+		for (int i = 1; i < 1000; i++) {
```

Comment:
Please indent (format the linebreaks) the file properly

Reply:
Also since this task is closely related to geometry and algebra, it worth the name the variables within those nomenclature. So please use either a, b, c or x, y, z instead if i, j ,k